### ISSUE TITLE
Append 'git describe' to ZFS_META_RELEASE

### ISSUE THREAD

**Issue by byteharmony** [zfsonlinux/spl#195]
When a system already has rc12 installed building a new master copy results in RPM files that the rpm system can not tell is different (newer) than the currently installed rc12 files.

As such # yum upgrade *64.rpm

fails with the error 
[root@nas100 rc12-master-20121201]# yum update *64.rpm
Loaded plugins: fastestmirror, refresh-packagekit, security
Determining fastest mirrors
- base: mirrors.gigenet.com
- extras: centos.mirrors.tds.net
- updates: mirror.sanctuaryhost.com
  base                                                     | 3.7 kB     00:00  
  extras                                                   | 3.5 kB     00:00  
  extras/primary_db                                        |  23 kB     00:00  
  updates                                                  | 3.5 kB     00:00  
  updates/primary_db                                       | 4.1 MB     00:00  
  Setting up Update Process
  Examining spl-0.6.0-rc12.x86_64.rpm: spl-0.6.0-rc12.x86_64
  spl-0.6.0-rc12.x86_64.rpm: does not update installed package.
  Examining spl-modules-0.6.0-rc12_2.6.32_279.14.1.el6.x86_64.rpm: spl-modules-0.6.0-rc12_2.6.32_279.14.1.el6.x86_64
  spl-modules-0.6.0-rc12_2.6.32_279.14.1.el6.x86_64.rpm: does not update installed package.
  Examining zfs-0.6.0-rc12.x86_64.rpm: zfs-0.6.0-rc12.x86_64
  zfs-0.6.0-rc12.x86_64.rpm: does not update installed package.
  Examining zfs-dracut-0.6.0-rc12.x86_64.rpm: zfs-dracut-0.6.0-rc12.x86_64
  zfs-dracut-0.6.0-rc12.x86_64.rpm: does not update installed package.
  Examining zfs-modules-0.6.0-rc12_2.6.32_279.14.1.el6.x86_64.rpm: zfs-modules-0.6.0-rc12_2.6.32_279.14.1.el6.x86_64
  zfs-modules-0.6.0-rc12_2.6.32_279.14.1.el6.x86_64.rpm: does not update installed package.
  Examining zfs-test-0.6.0-rc12.x86_64.rpm: zfs-test-0.6.0-rc12.x86_64
  zfs-test-0.6.0-rc12.x86_64.rpm: does not update installed package.
  No Packages marked for Update
  [root@nas100 rc12-master-20121201]# 

Running # yum reinstall *64.rpm
does appear to overwrite the packages, however a reboot is still needed and the resulting system has RPM packages installed that a user can not distinguish from the current release rc packages.

If we use that command then what is in use? the kernel version of the prior install package and the userland utils of the current install? How can one keep this straight to maintain the system?

Could we use the version number of the package to fix this?
In other words:
 zfs-0.6.12.1 - rc12 the first master release
 zfs-0.6.12.2 - rc12 the second master release
 zfs-0.6.12.3 - rc12 the third master release

if the 0.6.0 is somehow sacred, then dive deeper into the numbers:
0.6.0.12.1
